src/components/simpleareachart.js

```diff
@@ -11,75 +12,114 @@ import {
+  const handleShowModal = () => setShowDataModal(true)
   
   return (
-    <ResponsiveContainer className="simplearea-cont">
-      <AreaChart
-        width={800}
-        height={400}
-        data={data}
-        margin={{
-          top: 30,
-          right: 30,
-          left: 0,
-          bottom: 30
-        }}
-      >
-        {/* <Legend align="center" verticalAlign="top" iconType="square" iconSize="15" /> */}
-        <CartesianGrid strokeDasharray="3 3" />
-        <XAxis dataKey="year" >
-          <Label value="Year" offset={-15} position="insideBottom" />
-        </XAxis>
-        <YAxis />
-        <Tooltip />
-        <Area
-          type="monotone"
-          dataKey="hist"
-          stackId="1"
-          stroke="#b65c00"
-          fill="#e8ceb3"
-          name="Emissions"
-          isAnimationActive={false}
-        />
-        <Area
-          type="monotone"
-          dataKey="projected"
-          stackId="2"
-          stroke="#36a654"
-          fill="#c1e5cb"
-          name="Projection"
-          isAnimationActive={false}
-        />
-        <ReferenceDot y={dataMidPoint} x={currYear-4} stroke="none" fill="none" label={{ value: "Emissions", angle: 90, fill: "#b65c00" }} />
-        <ReferenceDot y={dataMidPoint} x={currYear+1} stroke="none" fill="none" label={{ value: "Projections", angle: 90, fill: "#36a654" }} />
-      </AreaChart>
-    </ResponsiveContainer>
+    <>
+      <ResponsiveContainer className="simplearea-cont">
+        <AreaChart
+          width={800}
+          height={400}
+          data={data}
+          margin={{
+            top: 30,
+            right: 30,
+            left: 0,
+            bottom: 30
+          }}
+        >
+          {/* <Legend align="center" verticalAlign="top" iconType="square" iconSize="15" /> */}
+          <CartesianGrid strokeDasharray="3 3" />
+          <XAxis dataKey={YearDataKey} >
+            <Label value="Year" offset={-15} position="insideBottom" />
+          </XAxis>
+          <YAxis />
+          <Tooltip />
+          <Area
+            type="monotone"
+            dataKey={EmissionsDataKey}
+            stackId="1"
+            stroke="#b65c00"
+            fill="#e8ceb3"
+            name="Emissions"
+            isAnimationActive={false}
+          />
+          <Area
+            type="monotone"
+            dataKey={ProjectionDataKey}
+            stackId="2"
+            stroke="#36a654"
+            fill="#c1e5cb"
+            name="Projection"
+            isAnimationActive={false}
+          />
+          <ReferenceDot y={dataMidPoint} x={currYear-4} stroke="none" fill="none" label={{ value: "Emissions", angle: 90, fill: "#b65c00" }} />
+          <ReferenceDot y={dataMidPoint} x={currYear+1} stroke="none" fill="none" label={{ value: "Projections", angle: 90, fill: "#36a654" }} />
+        </AreaChart>
+      </ResponsiveContainer>
+
+      <Button variant="secondary" onClick={handleShowModal}>
```

Comment:
what do you think about centering this button below the chart so it aligns with the xaxis label?